model_logging/migrations/0001_initial.py

```diff
@@ -1,8 +1,13 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
-
 from django.db import models, migrations
-import pgcrypto.fields
+from unittest.mock import Mock
+
+try:
+    import pgcrypto.fields
+except ImportError:
+    pgcrypto = Mock()
```

Comment:
Could i get a review on this please.

The assumption is that a user who is upgrading from 0.1.0 to this version will already have pgcrypto installed. We also check this in 0003 so that they have a verbose error message.

A new user shouldn't migrate through 0001 and should go through the squashed migrations. Hence we pretend that we have a pgcrypto object to allow the migration parsing at startup to be carried out.